Assets/MirageXR/Player/Scripts/Augmentations/AudioPlayer/AudioPlayer.cs

```diff
@@ -389,7 +387,19 @@ private IEnumerator LoadAudio()
                 AppLog.LogTrace("Trying to load audio: " + completeAudioName);
                 WWW www = new WWW(completeAudioName);
                 yield return www;
-                AudioClip audioClip = www.GetAudioClip(false, false, AudioType.WAV);
+                AudioClip audioClip;
+                if (filename.EndsWith("wav"))
+                {
+                    audioClip = www.GetAudioClip(false, false, AudioType.WAV);
+                }
+                else if (filename.EndsWith("mp3"))
+                {
+                    audioClip = www.GetAudioClip(false, false, AudioType.MPEG);
+                }
+                else
+                {
+                    audioClip = www.GetAudioClip(false, false, AudioType.UNKNOWN);
+                }
```

Comment:
move getting the type to a separate method.

and what happens if you try to load a file with type `AudioType.UNKNOWN`?
if it throws an error it needs to be handled.

```suggestion
                var audioType = GetAudioType(filename);
                AudioClip audioClip = www.GetAudioClip(false, false, audioType);
                
            private AudioType GetAudioType(string filename)
            {
                if (filename.EndsWith("wav"))
                {
                    return AudioType.WAV;
                }
                else if (filename.EndsWith("mp3"))
                {
                    return AudioType.MPEG;
                }
                else
                {
                    return AudioType.UNKNOWN;
                }
            }
```